package/utils/busybox/Makefile

```diff
@@ -93,9 +93,6 @@ LDLIBS:=m crypt
 endif
 
 LDLIBS += $(call BUSYBOX_IF_ENABLED,PAM,pam pam_misc pthread)
-ifeq ($(CONFIG_USE_GLIBC),y)
-  LDLIBS += $(call BUSYBOX_IF_ENABLED,NSLOOKUP_OPENWRT,resolv)
-endif
```

Comment:
I'm unsure about these lines. Should we also fallback to `!NSLOOKUP_BIG` if we use glibc?

Reply:
I wouldn't rather keep it consistent.